integrations/tensorboard/Neptune-TensorBoard.ipynb

```diff
@@ -0,0 +1,765 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "v1D-Y52T5OwT",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "# Neptune + TensorBoard"
```

Comment:
It would be cool to mention explicitly that we have 2 different ways of working in TB.